lib/checks/visibility/hidden-content.js

```diff
@@ -1,13 +1,16 @@
-let styles = window.getComputedStyle(node);
+const whitelist = ['SCRIPT', 'HEAD', 'TITLE', 'NOSCRIPT', 'STYLE', 'TEMPLATE'];
+if (
```

Comment:
I would like us to stick with an 80 char line length. My setup enlarged screen doesn't fit more than that on it. I also don't much like reading longer lines. They hare hard to scan. Anyway, I fixed the line, adding double indent.

We could consider something like Prettier, if we want to get a cleaner, more consistent format.